resources/public/api/conf/3.0/schemas/retrieve_property_bsas.json

```diff
@@ -1,154 +1,675 @@
   "type": "object",
-  "required": [
-    "metadata"
-  ],
   "properties": {
     "metadata": {
-      "$ref": "metadata_definitions.json#/definitions/propMetadata"
+      "type": "object",
+      "description": "Object containing the identifying data for the business and associated information for the BSAS with this ID.",
+      "properties": {
+        "calculationId": {
+          "description": "The unique identifier of the summary calculation.",
+          "type": "string",
+          "example": "717f3a7a-db8e-11e9-8a34-2a2ae2dbcce4"
+        },
+        "requestedDateTime": {
+          "description": "The date and time the summary calculation was originally requested, prior to any adjustments.",
+          "type": "string",
+          "example": "2019-04-04T01:01:01Z"
+        },
+        "adjustedDateTime": {
+          "description": "The date and time this calculation was adjusted.",
+          "type": "string",
+          "example": "2019-04-04T01:01:01Z"
+        },
+        "nino": {
+          "description": "National Insurance number, in the format AA999999A.",
+          "type": "string",
+          "example": "TC663795B"
+        },
+        "taxYear": {
+          "description": "The tax year the data applies to, for example, 2021-22. The start year and end year must not span two tax years. No gaps are allowed, for example, 2020-22 is not valid.",
+          "type": "string",
+          "example": "2021-22"
+        },
+        "summaryStatus": {
+          "type": "string",
+          "enum": [
+            "valid",
+            "invalid",
+            "superseded"
+          ],
+          "description": "The status of the this calculated summary, limited to the following possible values: \n\n valid \n\n invalid \n\n superseded"
+        }
+      },
+      "required": [
+        "calculationId",
+        "requestedDateTime",
+        "nino",
+        "taxYear",
+        "summaryStatus"
+      ]
     },
-    "bsas": {
+    "inputs": {
       "type": "object",
-      "description": "Object holding the BSAS for either a UK property non-Furnished Holiday Let (non-FHL) or for a UK Property Furnished Holiday Let (FHL) business income source.",
+      "description": "Object containing the inputs that were used in this calculation.",
       "properties": {
-        "total": {
-          "$ref": "definitions.json#/definitions/bsasTotal"
+        "businessId": {
+          "description": "An identifier for the business, unique to the customer. Must conform to the regular expression: ^X[A-Z0-9]{1}IS[0-9]{11}$",
+          "type": "string",
+          "example": "XAIS12345678910"
+        },
+        "typeOfBusiness": {
+          "type": "string",
+          "enum": [
+            "uk-property-fhl",
+            "uk-property-non-fhl"
+          ],
+          "description": "The type of business the calculation summary is for. Limited to the following possible values: \n\n uk-property-fhl \n\n uk-property-non-fhl"
```

Comment:
You don't need to include `Limited to the following possible values: \n\n uk-property-fhl \n\n uk-property-non-fhl` in the description because by assigning it as an enum it will automatically include this line. When you include it in the description it duplicates on the documentation